packages/task/src/browser/task-configurations.ts

```diff
@@ -44,11 +45,14 @@ export class TaskConfigurations implements Disposable {
 
     protected readonly toDispose = new DisposableCollection();
     /**
-     * Map of source (path of root folder that the task config comes from) and task config map.
+     * Map of source (path of root folder that the task configs come from) and task config map.
      * For the inner map (i.e., task config map), the key is task label and value TaskConfiguration
      */
     protected tasksMap = new Map<string, Map<string, TaskConfiguration>>();
-    protected taskCustomizations: TaskCustomization[] = [];
+    /**
+     * Map of source (path of root folder that the task configs come from) and task customizations map.
+     */
```

Comment:
The function header line 48 is the same as the comment inside the method (line 52-54)
This is duplication info in a few lines, may be just one would be enough.

Reply:
It's not actually the same, one is `TaskConfiguration` while the other is `TaskCustomization`.